addons/mrp/migrations/10.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,419 @@
+mrp          / mrp.unbuild              / state (selection)             : NEW selection_keys: ['done', 'draft']
+
+
+mrp          / mrp.workcenter           / capacity_per_cycle (float)    : was renamed to capacity [nothing to do]
+mrp          / mrp.workcenter           / color (integer)               : NEW
+mrp          / mrp.workcenter           / costs_cycle (float)           : DEL
+mrp          / mrp.workcenter           / costs_cycle_account_id (many2one): DEL relation: account.analytic.account
+mrp          / mrp.workcenter           / costs_general_account_id (many2one): DEL relation: account.account
+mrp          / mrp.workcenter           / costs_hour (float)            : DEL
+mrp          / mrp.workcenter           / costs_hour_account_id (many2one): DEL relation: account.analytic.account
+mrp          / mrp.workcenter           / oee_target (float)            : NEW
+mrp          / mrp.workcenter           / order_ids (one2many)          : NEW relation: mrp.workorder
+mrp          / mrp.workcenter           / product_id (many2one)         : DEL relation: product.product
+mrp          / mrp.workcenter           / routing_line_ids (one2many)   : NEW relation: mrp.routing.workcenter
+mrp          / mrp.workcenter           / time_cycle (float)            : DEL
+mrp          / mrp.workcenter           / time_ids (one2many)           : NEW relation: mrp.workcenter.productivity
+# NOTHING TO DO
+
+mrp          / mrp.workcenter           / sequence (integer)            : NEW required: required, req_default: function
+# DONE: Used the default value
+
+
+##### THIS IS A NEW MODEL. NOTHING TO DO #######
+mrp          / mrp.workcenter.productivity / date_end (datetime)           : NEW
+mrp          / mrp.workcenter.productivity / date_start (datetime)         : NEW required: required, req_default: function
+mrp          / mrp.workcenter.productivity / description (text)            : NEW
+mrp          / mrp.workcenter.productivity / loss_id (many2one)            : NEW relation: mrp.workcenter.productivity.loss, required: required
+mrp          / mrp.workcenter.productivity / user_id (many2one)            : NEW relation: res.users
+mrp          / mrp.workcenter.productivity / workcenter_id (many2one)      : NEW relation: mrp.workcenter, required: required
+mrp          / mrp.workcenter.productivity / workorder_id (many2one)       : NEW relation: mrp.workorder
+mrp          / mrp.workcenter.productivity.loss / loss_type (selection)         : NEW required: required, selection_keys: ['availability', 'performance', 'productive', 'quality'], req_default: function
+mrp          / mrp.workcenter.productivity.loss / manual (boolean)              : NEW
+mrp          / mrp.workcenter.productivity.loss / name (char)                   : NEW required: required
+mrp          / mrp.workcenter.productivity.loss / sequence (integer)            : NEW
+
+####### DONE: Renamed table 'mrp_production_workcenter_line' to 'mrp_workorder' ################
```

Comment:
Do all fields match one by one?

Reply:
Some fields come from the <code>mrp.operations</code> module and some other fields are new. But there are many matches.

Reply:
Please double check if any of the fields has been renamed

Reply:
Yes, I checked them :wink:, that's why in the <code>_field_renames</code> I put

            ('mrp.production', 'mrp_production', 'date_planned', 'date_planned_start'),

Reply:
Then reflect it here on the work file